pkg/networkmanager/firewall.jsx

```diff
@@ -114,43 +117,62 @@ class SearchInput extends React.Component {
     render() {
         return <input autoFocus
                       id={this.props.id}
+                      value={this.state.value}
```

Comment:
Curious that this worked before -- with React's [controlled components](https://reactjs.org/docs/forms.html) I thought this pretty much always needs to happen. But this is an unrelated bug fix, can you please split this out into a separate commit?

Reply:
It worked because parent object is storing the value for itself (and does then filtering) and this component itself did not need to store it explicitly. I needed to introduce it because if you write something into the search bar and then go to `custom ports` and then back the search bar would be empty while the previous value was still filtering. But I will create it as separate commit as it makes sense on its own.